ee/app/livechat-enterprise/server/hooks/checkDefaultAgentOnNewRoom.js

```diff
@@ -0,0 +1,36 @@
+import { callbacks } from '../../../../../app/callbacks';
+import { RoutingManager } from '../../../../../app/livechat/server/lib/RoutingManager';
+import { settings } from '../../../../../app/settings';
```

Comment:
we're already removing the index files from modules root folders, so please always import at least from `server` or `client` folders
```suggestion
import { callbacks } from '../../../../../app/callbacks/server';
import { RoutingManager } from '../../../../../app/livechat/server/lib/RoutingManager';
import { settings } from '../../../../../app/settings/server';
```